lib/internal/console/constructor.js

```diff
@@ -509,12 +509,12 @@ const consoleMethods = {
   },
 };
 
-// Returns true if label was not found
+// Returns true if label was found
 function timeLogImpl(self, name, label, data) {
   const time = self._times.get(label);
-  if (!time) {
+  if (time === undefined) {
     process.emitWarning(`No such label '${label}' for console.${name}()`);
-    return true;
+    return false;
```

Comment:
Is this right?

Reply:
Yes. I intentionally switched the return value.